documentation/processing.tex

```diff
@@ -52,13 +54,13 @@ \subsection{The main loop of manager}
 
 \begin{enumerate}
 	\item When a processor finishes its work, the next processor is selected from the array.
-	\item When a provider finishes --- before the next provider is selected from the array --- manager checks whether it triggers the termination of the current processor as well (see \emph{terminal condition} in \cref{lab06:sect_proc}). If true, perform rule 1.
+	\item When a provider finishes --- before the next provider is selected from the array --- manager checks whether it triggers the termination of the current processor as well (see \emph{terminal condition} in \cref{lab06:sect_proc}). Only if true, perform rule 1.
 \end{enumerate}
 
 \subsection{Initial state of manager}
 \label{lab06:lib_data}
 
-It is important which statement provider and processor is assigned to the manager as a default. The manager determines this from \emph{library data} passed by analyzer.
+During initialization, the manager can set various statement providers and processors as a default. It is very important as it determines the way how the source is processed. The manager determines this from \emph{library data} passed by analyzer.
```

Comment:
```suggestion
During initialization, the manager sets various statement providers and processors as a default. It is very important as it determines the way how the source is processed. The manager determines this from \emph{library data} passed by analyzer.
```

Reply:
preco?
teraz to znie akoby ich vedel viac setnut ako default naraz

Reply:
z tohto pohladu tie 2 vety hovoria podla mna to iste